org.eclipse.m2e.pde.ui/src/org/eclipse/m2e/pde/ui/editor/MavenTargetLocationWizard.java

```diff
@@ -261,21 +307,21 @@ private CCombo combo(CCombo combo) {
 	@Override
 	public boolean canFinish() {
 		if (isCreateFeature()) {
-			return page.isPageComplete();
+			return super.canFinish();
```

Comment:
Is this and the changes in the following lines related to the inclusion-improvements or is it issue #457?

Reply:
Actually the `isCreateFeature()` was inverse logic (returned false if a feature has to be created) as it has confused me I refactored it along with this change.